components/files/configurationmanager.hpp

```diff
@@ -88,6 +88,17 @@ struct escape_hash_filter : public boost::iostreams::input_filter
         bool mFinishLine;
 };
 
+struct unescape_hash_filter : public boost::iostreams::input_filter
+{
+    unescape_hash_filter();
+    virtual ~unescape_hash_filter();
+    
+    template <typename Source> int get(Source & src);
```

Comment:
I would guess that your template definition is causing some issues. Maybe try moving your implementation to the header file.